llvm/include/llvm/Support/Chrono.h

```diff
@@ -33,13 +33,33 @@ namespace sys {
 template <typename D = std::chrono::nanoseconds>
 using TimePoint = std::chrono::time_point<std::chrono::system_clock, D>;
 
+// utc_clock and utc_time are only available since C++20. Add enough code to
+// support formatting date/time in UTC.
+class UtcClock : public std::chrono::system_clock {};
```

Comment:
This Support change needs to be split from this patch with a unittest in llvm/unittest/Support.

Many people will think this z/OS patch is unrelated to them.

Reply:
I created a new PR for the UTC clock changes here: https://github.com/llvm/llvm-project/pull/67846